docroot/themes/humsci/humsci_basic/src/scss/utilities/_general.scss

```diff
@@ -64,6 +64,7 @@
     @include hb-global-color('background-color', 'gray-light');
   }
 
+  padding: hb-calculate-rems(20px) hb-calculate-rems(18px);
```

Comment:
do you think `hb-borderless` should get this padding too? I wasn't sure

Reply:
@meranicosme would like for the content with `hb-borderless` to align with the rest of the content on the page. Only groups with a background or border should get padding.